doc/source/index.md

```diff
@@ -67,7 +67,7 @@
         <p style="color:#515151;">Understand how the Ray framework scales your ML workflows.</p>      
         <p style="font-weight:600;">Learn more > </p>  
   </div> </a>  
-   <a class="no-underline" href="./ray-overview/installation.html" target="_blank"> <div class="info-box">
+   <a class="no-underline" href="./ray-overview/installation.html" target="_blank"> <div class="info-box" style="height:100%;">
```

Comment:
nit: Would it make sense to apply this to the other cards as well to guard against future inconsistencies?

Reply:
Thanks, @matthewdeng 
Made a commit to fix this.